packages/terra-paginator/src/ProgressivePaginator.jsx

```diff
@@ -6,7 +6,7 @@ import ResponsiveElement from 'terra-responsive-element';
 import 'terra-base/lib/baseStyles';
 import styles from './Paginator.module.scss';
 
-import { calculatePages, KEYCODES } from './_paginationUtils';
+import { calculatePages, KeyCode } from './_paginationUtils';
```

Comment:
I feel like we should directly import from the keycode package and remove the keycode exports from `_paginationUtils.js`